### ISSUE TITLE
Model Integration Request for medigan 00100_WGANGP_MMG_MASS_ROI

### ISSUE THREAD

**Issue by RichardObi** [RichardObi/medigan#21]
Creator: test | Affiliation: test affiliation | Description: test description 
 Model:00100_WGANGP_MMG_MASS_ROI 
 Package stored in: /Users/richardosuala/Desktop/00100_WGANGP_MMG_MASS_ROI 
Model Metadata: 
{
   "00100_WGANGP_MMG_MASS_ROI": {
      "execution": {
         "package_name": "WGANGP_MMG_MASSES_BCDR_IWBI22",
         "package_link": "/Users/richardosuala/Desktop/00100_WGANGP_MMG_MASS_ROI",
         "model_name": "10000",
         "extension": ".pt",
         "image_size": [
            128,
            128
         ],
         "dependencies": [
            "numpy",
            "torch",
            "opencv-contrib-python-headless"
         ],
         "generate_method": {
            "name": "generate",
            "args": {
               "base": [
                  "model_file",
                  "num_samples",
                  "output_path",
                  "save_images"
               ],
               "custom": {}
            }
         }
      },
      "selection": {
         "performance": {
            "SSIM": null,
            "MSE": null,
            "NSME": null,
            "PSNR": null,
            "IS": null,
            "FID": null,
            "turing_test": null,
            "downstream_task": {
               "CLF": {
                  "trained_on_fake": {
                     "accuracy": 0.95,
                     "f1": 0.969,
                     "AUROC": 0.978,
                     "AUPRC": 0.996
                  }
               }
            }
         },
         "use_cases": [
            "classification"
         ],
         "organ": [
            "breast",
            "breasts",
            "chest"
         ],
         "modality": [
            "MMG",
            "Mammography",
            "Mammogram",
            "full-field digital",
            "full-field digital MMG",
            "full-field MMG",
            "full-field Mammography",
            "digital Mammography",
            "digital MMG",
            "x-ray mammography"
         ],
         "vendors": [],
         "centres": [],
         "function": [
            "noise to image",
            "image generation",
            "unconditional generation",
            "data augmentation"
         ],
         "condition": [],
         "dataset": [
            "BCDR"
         ],
         "augmentations": [
            "horizontal flip",
            "vertical flip"
         ],
         "generates": [
            "mass",
            "masses",
            "mass roi",
            "mass ROI",
            "mass images",
            "mass region of interest",
            "nodule",
            "nodule",
            "nodule roi",
            "nodule ROI",
            "nodule images",
            "nodule region of interest"
         ],
         "height": 128,
         "width": 128,
         "depth": null,
         "type": "WGAN-GP",
         "license": "MIT",
         "dataset_type": "public",
         "privacy_preservation": null,
         "tags": [
            "Breast",
            "Mammogram",
            "Mammography",
            "Digital Mammography",
            "Full field Mammography",
            "Full-field Mammography",
            "128x128",
            "128 x 128",
            "MammoGANs",
            "Masses",
            "Nodules"
         ],
         "year": "2022"
      },
      "description": {
         "title": "WGAN-GP Model for Mammogram MASS Patch Generation (Trained on BCDR)",
         "provided_date": "Mar 2022",
         "trained_date": "Mar 2022",
         "provided_after_epoch": 10000,
         "version": "1.0.0",
         "publication": null,
         "doi": [
            "10.48550/arXiv.2203.04961"
         ],
         "comment": "A wasserstein generative adversarial network with gradient penalty (WGAN-GP) that generates mass patches of mammograms. Pixel dimensions are 128x128. The DCGAN was trained on MMG patches from the BCDR dataset (Lopez et al, 2012). The uploaded ZIP file contains the files 10000.pt (model weight), __init__.py (image generation method and utils), a requirements.txt, and the GAN model architecture (in pytorch) below the /src folder."
      }
   }
}
